omod/src/main/resources/config.xml

```diff
@@ -9,101 +9,112 @@
     graphic logo is a trademark of OpenMRS Inc.
 -->
 <!DOCTYPE module PUBLIC "-//OpenMRS//DTD OpenMRS Config 1.0//EN"
```

Comment:
Is there anyway to not change all the white space in the file as part of this PR?

Reply:
Yeah, I can do it.